playwright-e2e/lib/component/Question.ts

```diff
@@ -100,6 +101,7 @@ export default class Question {
     const isChecked = await Question.isChecked(loc);
     if (!isChecked) {
       await loc.click();
+      await expect(loc).toHaveClass(/checkbox-checked|radio-checked/);
```

Comment:
UI synchronization tricks: Checks and waits (up to 10 sec) until checkbox/radiobutton is visually checked.